conanfile.py

```diff
@@ -1,76 +1,128 @@
-from conans import ConanFile, CMake, tools
 import os
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.53.0"
```

Comment:
Just to be sure: this conanfile is still compatible with conan 1?

Reply:
Yes. At the moment, the conanfile is written in such a way that it is backwards-compatible with conan 1.x.